### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestions

### CODE REVIEW

File: pages/product/product.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+/* ---------------------card container-------------- */
+.card-container {
+    width: 100%;
+    height: 81vh;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 20px 15px;
```
Comment (Jia-A) [inline pages/product/product.css:75]: You can change px to rem here.

**Review by Jia-A — APPROVED** [review]
Good to merge. Just make a different css file for flex properties and use it.

**Review by Sen-442b — APPROVED** [review]
Looks good to me

File: pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+.horizontal .card-header-img img {
+    border-top-right-radius: unset;
+}
+
+.card-description {
+    width: 50%;
+}
+.product-qty{
+    margin: 0.5rem 0 0.5rem 0;
```
Comment (SaurabhChawda) [inline pages/cart/cart.css:55]: Use Shorthand Property.

File: pages/cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+.qty-btn{
+    background-color: var(--secondary-grey);
+    border: none;    
+    border-radius: 50%;
+    padding: 0.1rem 0rem 0.1rem 0rem;
```
Comment (SaurabhChawda) [inline pages/cart/cart.css:64]: Here also.

**Review by SaurabhChawda — APPROVED** [review]

**Pull request merged**